### DISCUSSION TITLE
react-native error after import firebase

### DISCUSSION THREAD

**Discussion by reactdeveloper22** [reactdeveloper22/Instagram-Clone#1] [Q&A] [answered]
I have installed firebase package in my project using cmd
```
npm install --save firebase 
```
When I import firebase to my react-native project import firebase from 'firebase' I'm getting this error. What could be the problem?

**Comment by SashenJayathilaka** [comment, marked as answer]
### After installation
firebase import
```
import firebase from "firebase/compat/app";
import "firebase/compat/auth";
import "firebase/compat/firestore";
```
#### or ``npm i firebase@9.6.11``
